src/main/java/com/scriptbasic/utility/functions/UtilityFunctions.java

```diff
@@ -143,4 +146,19 @@ public static Long length(final Object arg) {
         return null;
     }
 
+    @BasicFunction(classification = Utility.class)
+    public static Double cdbl(final Object arg) throws BasicRuntimeException {
+        if (arg == null) {
+            throw new BasicRuntimeException("NULL cannot be converted to double");
```

Comment:
The exception text talks to the BASIC programmer. They do not know `NULL`. As a matter of fact, it is `null` even in Java. It is `NULL` in C++.

For the ScriptBasic developer it is `undef`. Please change the message so that this is meaningful for the BASIC developer.